{
	"document_title": "CVE-2023-49297",
	"document_type": "SUSE CVE",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "SUSE CVE-2023-49297"
		},
		"status": "Interim",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "3",
				"date": "2023-12-13T00:09:31Z",
				"description": "current"
			}
		},
		"initial_release_date": "2023-12-07T00:09:43Z",
		"current_release_date": "2023-12-13T00:09:31Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf-cve.pl",
			"date": "2020-12-27T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "CVE-2023-49297",
			"title": "CVE",
			"type": "Summary"
		},
		{
			"text": "PyDrive2 is a wrapper library of google-api-python-client that simplifies many common Google Drive API V2 tasks. Unsafe YAML deserilization will result in arbitrary code execution. A maliciously crafted YAML file can cause arbitrary code execution if PyDrive2 is run in the same directory as it, or if it is loaded in via `LoadSettingsFile`. This is a deserilization attack that will affect any user who initializes GoogleAuth from this package while a malicious yaml file is present in the same directory. This vulnerability does not require the file to be directly loaded through the code, only present. This issue has been addressed in commit `c57355dc` which is included in release version `1.16.2`. Users are advised to upgrade. There are no known workarounds for this vulnerability.",
			"title": "Mitre CVE Description",
			"type": "Description"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		}
	],
	"document_references": [
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		}
	],
	"product_tree": {
		"branch": [
			{
				"type": "Product Family",
				"name": "openSUSE Tumbleweed",
				"branch": [
					{
						"type": "Product Name",
						"name": "openSUSE Tumbleweed",
						"full_product_name": {
							"text": "openSUSE Tumbleweed",
							"product_id": "openSUSE Tumbleweed",
							"cpe": "cpe:/o:opensuse:tumbleweed"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Version",
				"name": "python310-pydrive2-1.16.2-1.1",
				"full_product_name": {
					"text": "python310-pydrive2-1.16.2-1.1",
					"product_id": "python310-pydrive2-1.16.2-1.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python311-pydrive2-1.16.2-1.1",
				"full_product_name": {
					"text": "python311-pydrive2-1.16.2-1.1",
					"product_id": "python311-pydrive2-1.16.2-1.1"
				}
			},
			{
				"type": "Product Version",
				"name": "python39-pydrive2-1.16.2-1.1",
				"full_product_name": {
					"text": "python39-pydrive2-1.16.2-1.1",
					"product_id": "python39-pydrive2-1.16.2-1.1"
				}
			}
		],
		"relationship": [
			{
				"product_reference": "python310-pydrive2-1.16.2-1.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"product_reference": "python311-pydrive2-1.16.2-1.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"product_reference": "python39-pydrive2-1.16.2-1.1",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			}
		]
	},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "PyDrive2 is a wrapper library of google-api-python-client that simplifies many common Google Drive API V2 tasks. Unsafe YAML deserilization will result in arbitrary code execution. A maliciously crafted YAML file can cause arbitrary code execution if PyDrive2 is run in the same directory as it, or if it is loaded in via `LoadSettingsFile`. This is a deserilization attack that will affect any user who initializes GoogleAuth from this package while a malicious yaml file is present in the same directory. This vulnerability does not require the file to be directly loaded through the code, only present. This issue has been addressed in commit `c57355dc` which is included in release version `1.16.2`. Users are advised to upgrade. There are no known workarounds for this vulnerability.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2023-49297",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"openSUSE Tumbleweed:python310-pydrive2-1.16.2-1.1",
						"openSUSE Tumbleweed:python311-pydrive2-1.16.2-1.1",
						"openSUSE Tumbleweed:python39-pydrive2-1.16.2-1.1"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "important"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {},
				"score_set_v_3": {
					"base_score_v_3": "7.8",
					"vector_v_3": "CVSS:3.1/AV:L/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H"
				}
			}
		}
	]
}
